include/alph_puzzle.h

```diff
@@ -18,6 +18,16 @@ typedef enum AlphPuzzles {
     ALPH_PUZZLE_MAX
 } AlphPuzzles;
 
+typedef struct UnkAlphSub_10 {
+    void *unk_00; // unused
```

Comment:
unk -> unused/filler/something else in struct field names

Reply:
Deferring to #290